x-pack/plugin/src/test/resources/rest-api-spec/api/transform.cat_transform.json

```diff
@@ -1,5 +1,5 @@
 {
-  "transform.cat_transform":{
+  "cat.transform":{
```

Comment:
I think you also need to change the file name to match this

Reply:
:doh: renamed: `x-pack/plugin/**out**/test/resources/rest-api-spec/api/cat.transform.json` but missed this one. thanks @droberts195